Tests/TestingApps/PaywallsTester/PaywallsTester/Views/AppContentView.swift

```diff
@@ -58,11 +59,6 @@ struct AppContentView: View {
                     }
             }
         }
-        .presentPaywallIfNeeded {
-            !$0.hasPro
```

Comment:
Are we testing this somewhere else? To make sure it works as expected, not sure if we could move it somewhere else...

Reply:
gonna to it as a separate PR